src/chat.rs

```diff
@@ -3737,7 +3737,7 @@ async fn rename_ex(
     if !success {
         bail!("Failed to set name");
     }
-    if sync.into() {
+    if sync.into() && chat.name != new_name {
         let sync_name = sync_name.to_string();
```

Comment:
Maybe then we should send `new_name` instead here, but i think it's minor, `improve_single_line_input()` logic shouldn't change often

Reply:
Yes, it's exactly what i decided to do also: #5021